Cargo.toml

```diff
@@ -42,7 +41,7 @@ mysql_async = { version = "0.34.1", features = ["native-tls-tls", "chrono"], opt
 prost = { version = "0.12" , optional = true } # pinned for arrow-flight compat
 r2d2 = { version = "0.8.10", optional = true }
 rusqlite = { version = "0.31.0", optional = true }
-sea-query = { version = "0.31.0", features = ["backend-sqlite", "backend-postgres", "postgres-array", "with-rust_decimal", "with-bigdecimal", "with-time", "with-chrono"] }
+sea-query = { git = "https://github.com/spiceai/sea-query.git", rev = "213b6b876068f58159ebdd5852604a021afaebf9", features = ["backend-sqlite", "backend-postgres", "postgres-array", "with-rust_decimal", "with-bigdecimal", "with-time", "with-chrono"] }
```

Comment:
This will prevent me from publishing a new version of this crate

Reply:
I will try to replace it with crate version